packages/runtime/src/helpers/edge.ts

```diff
@@ -85,6 +92,7 @@ import {
 // Deno defines "window", but naughty libraries think this means it's a browser
 delete globalThis.window
 globalThis.process = { env: {...Deno.env.toObject(), NEXT_RUNTIME: 'edge', 'NEXT_PRIVATE_MINIMAL_MODE': '1' } }
+globalThis.EdgeRuntime = "netlify-edge"
```

Comment:
This is a "version" string. It's set to "edge-runtime" in the open source Next.js, and "vercel" when deployed to Vercel.

Reply:
For additional context, see https://github.com/vercel/edge-runtime/blob/fbf30d4b896d81b0c6508818e177d7f7d5c5cac4/docs/pages/features/available-apis.mdx#addressing-the-runtime